milv.config.yaml

```diff
@@ -1,6 +1,2 @@
-white-list-external: ["localhost", "console.kyma.local"]
-black-list: ["/vendor/"]
-files:
-  - path: "./examples/call-ec/web-ui/README.md"
-    config:
-      white-list-internal: ["../config/webpack.config.prod.js"]
\ No newline at end of file
+white-list-external: ["localhost", "kyma.local", "kyma-system"]
+black-list: ["/vendor/", "./examples/call-ec/web-ui/README.md"]
```

Comment:
Why `./examples/call-ec/web-ui/README.md` is added to `black-list`?

Reply:
This is React template from `Create React App` so I decide that checking links in this file is unnecessary. Updating those links would be like updating links in create-react-app repo.

Reply:
Ok, add proper comment to `milv.config.yaml`.